test/verify/check-packagekit

```diff
@@ -231,6 +240,179 @@ class TestUpdates(NoSubManCase):
         self.assertEqual(b.attr(self.update_icon, "class"), "fa fa-check-circle-o")
         b.wait_text(self.update_text, "System is up to date")
 
+    def testTracer(self):
+        b = self.browser
+        m = self.machine
+
+        class Tracer(object):
+            def __init__(self, testObj, rebootRequired=False, testStatusCard=False, packageName="vanilla", appName="foo", serviceName="slumber"):
```

Comment:
appName and serviceName don't get set explicitly, so I suggest to remove these arguments for now. Let's add them if/when needed (but I doubt that the tests ever need to customize them)